include/zephyr/bluetooth/services/ots.h

```diff
@@ -888,6 +902,19 @@ struct bt_ots_client_cb {
 	 */
 	void (*obj_data_written)(struct bt_ots_client *ots_inst,
 				 struct bt_conn *conn, size_t len);
+
+	/** @brief Callback function when checksum indication is received.
+	 *
+	 *  Called when the oacp_ind_handler received response of
+	 *  OP BT_GATT_OTS_OACP_PROC_CHECKSUM_CALC.
+	 *
+	 *  @param ots_inst          Pointer to the OTC instance.
+	 *  @param conn              The connection to the peer device.
+	 *  @param err               Error code (bt_ots_olcp_res_code).
+	 *  @param checksum          Checksum code (bt_ots_olcp_res_code).
```

Comment:
I assume that `checksum` will be `0` if `err != 0`?

Reply:
Modify description to 
`	 *  @param checksum          Checksum if error code is BT_GATT_OTS_OACP_RES_SUCCESS,
	 *                           otherwise 0.`